consensus/ethash/consensus.go

```diff
@@ -635,6 +638,9 @@ func (ethash *Ethash) SealHash(header *types.Header) (hash common.Hash) {
 	if header.BaseFee != nil {
 		enc = append(enc, header.BaseFee)
 	}
+	if header.WithdrawalsHash != nil {
+		enc = append(enc, header.WithdrawalsHash)
```

Comment:
Ethash(pre-merger) should have no withdrawal notion at all?

Reply:
panics now